### PR TITLE
feat: Add support for sending stickers

### PR DESCRIPTION
**Pull request by Nanrech**
## About

This pull request makes it possible for users to send stickers to a channel.

## Checklist

- [x] I've ran `pre-commit` to format and lint the change(s) made.
- [x] I've checked to make sure the change(s) work on `3.8.6` and higher.
- [x] This fixes/solves an [Issue](https://github.com/goverfl0w/discord-interactions/issues).
  - (If existent):
- [x] I've made this pull request for/as: (check all that apply)
  - [x] Documentation
  - [ ] Breaking change
  - [x] New feature/enhancement
  - [ ] Bugfix

### CODE REVIEW

**Comment by Nanrech** [comment]
PR should be ready.

File: interactions/api/http/message.py [hunk 1]
```diff
@@ -54,6 +55,9 @@ async def send_message(
         if message_reference:
             payload["message_reference"] = message_reference
 
+        if stickers:
+            payload["sticker_ids"] = [sticker.id for sticker in stickers]
```
Comment (sillyElvy) [inline interactions/api/http/message.py]: doesn't it need to be int/str?

**Review by sillyElvy — CHANGES_REQUESTED** [review]

File: interactions/api/http/message.py (hunk 1 above)
Comment (Nanrech) [reply interactions/api/http/message.py]: It needs to be a string.

Comment (sillyElvy) [reply interactions/api/http/message.py]: Then change it please, also you got merge conflict

Comment (Nanrech) [reply interactions/api/http/message.py]: Fixed the string part.

Tried fixing the merge conflict. Is it ok now?

Comment (sillyElvy) [reply interactions/api/http/message.py]: Looks like, yeah

**Review by sillyElvy — CHANGES_REQUESTED** [review]
Please add stickers to edit and reply in message.py

**Review by sillyElvy — APPROVED** [review]

**Pull request merged**